lib/__tests__/does-not-match-selector.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-env jest */
+
+import TestAssertions from '../helpers/test-assertions';
+
+describe('assert.dom(...).doesNotMatchSelector()', () => {
+  let assert;
+
+  beforeEach(() => {
+    assert = new TestAssertions();
+    document.body.innerHTML = '<div><p class="first"></p><p></p><p class="last"></p></div>';
+  });
+
+  describe('success scenarios', () => {
+    test('one element (selector passed) not matching compareSelector', () => {
+      assert.dom('p.first').doesNotMatchSelector('p + p');
+
+      expect(assert.results).toEqual([{
+        actual: 0,
+        expected: 0,
```

Comment:
we should use better `actual/expected` values. have a look at some of the other assertions, we often reuse the assertion message, and generate another assertion message for `expected`.

Reply:
I have been coming down with severe health problems during the weekend. I will get to it once I am well enough to code again. Sorry for the delay.

Reply:
no worries, gute besserung 😉

Reply:
thanks very much.